samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -2415,12 +2415,10 @@ public String getSelectURL(){
   
 	private Site getCurrentSite(String id) {
 		Site site = null;
-		//Placement placement = ToolManager.getCurrentPlacement();
-		//String currentSiteId = placement.getContext();
 		try {
 			site = SiteService.getSite(id);
 		} catch (IdUnusedException e) {
-			log.error(e.getMessage(), e);
+			log.error("Site not found, possibly anonymous assessment");
```

Comment:
If this will remain categorized as an error, then having more information about the error is of value:

log.error("Site not found for id {}; possibly anonymous assessment: {}", id, e.getMessage());

Reply:
sure, please edit PR direct or use the suggestion button